zmsadmin/src/Zmsadmin/WorkstationSelect.php

```
@@ -26,6 +27,9 @@ public function readResponse(
         if (!$workstation->hasId()) {
             return \BO\Slim\Render::redirect('index', array('error' => 'login_failed'));
         }
+        if ($wrongModuleResponse = ModuleAccess::rejectWrongModuleAccess(ModuleAccess::MODULE_ADMIN, $workstation, $response)) {
+            return $wrongModuleResponse;
```

PHPStan (2.1.54)

[error] Method BO\Zmsadmin\WorkstationSelect::readResponse() should return string but returns Psr\Http\Message\ResponseInterface.


(return.type)